### ISSUE TITLE
Wrong behavior with function calc-no-unspaced-operator using sass var

### ISSUE THREAD

**Issue by rmoralp** [stylelint/stylelint#6063]
### What steps are needed to reproduce the bug?

When `function-calc-no-unspaced-operator` is configured as true
Given we use a dash separated sass variables and in some cases we add them to a `calc`
Then _stylelint_ advise us incorrectly that it is an error, identifying it as an operation and not as a sass variable

Example:
```scss
$example-var: 2px;

a { font-size: calc(#{$example-var} * 2); }
```

Error:
<img width="733" alt="image" src="https://user-images.githubusercontent.com/5390428/166250716-6c16b529-fb0c-4f95-bdd7-e88747863388.png">

### What Stylelint configuration is needed to reproduce the bug?

{
  "rules": {
    "function-calc-no-unspaced-operator": true
  }
}

### How did you run Stylelint?

stylelint "**/*.scss" --config myconfig.json

### Which version of Stylelint are you using?

14.8.1

### What did you expect to happen?

Not to get an error

### What actually happened?

It throws the errors  "Expected single space before "-" operator" and  "Expected single after before "-" operator"

### Does the bug relate to non-standard syntax?

Yes, its related to SCSS

**Comment by ybiquitous** [comment]
@rmoralp Thanks for opening the report and using the template.

This issue was fixed by #6053. I'll release a patch version in a few days.

Duplicate of #6052
